engines/wintermute/detection_tables.h

```diff
@@ -376,7 +378,15 @@ static const WMEGameDescription gameDescriptions[] = {
 	WME_WINENTRY("dreaming", "",
 		WME_ENTRY1s("data.dcp", "4af26d97ea063fc1277ce30ae431de90", 8804073), Common::EN_ANY, ADGF_UNSTABLE, LATEST_VERSION),
 
-	// Dreamcat
+	// The Driller Incident (English)
```

Comment:
Please fix spacing here (that was me, I tried to fix the merge conflicts, and I missed the spacing)

Reply:
Fixed at 363be76bef8cf852823ed141fc83b05fc76af893